<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.545, x2:0.977, y2:0.539773, stop:0 rgba(65, 51, 83, 255), stop:1 rgba(34, 27, 44, 255));</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>771</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 245, 237);</string>
       </property>
       <property name="text">
        <string>CREATE NEW ACCOUNT</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_appname">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>0</y>
         <width>201</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 107, 241);
background-color:none</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;ELITE&lt;/strong&gt; ATM </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>140</y>
         <width>781</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 245, 237);</string>
       </property>
       <property name="text">
        <string>Fill in the form below with your details </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_2_accountno">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>260</y>
         <width>200</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 170, 255);
background-color:none</string>
       </property>
       <property name="text">
        <string>ACCOUNT TYPE :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2_accountno_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>290</y>
         <width>200</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 170, 255);
background-color:none</string>
       </property>
       <property name="text">
        <string>NAME</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2_accountno_3">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>350</y>
         <width>200</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 170, 255);
background-color:none</string>
       </property>
       <property name="text">
        <string>EMAIL ADDRESS:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2_accountno_4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>320</y>
         <width>200</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 170, 255);
background-color:none</string>
       </property>
       <property name="text">
        <string>PHONE NO :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2_accountno_6">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>380</y>
         <width>200</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 170, 255);
background-color:none</string>
       </property>
       <property name="text">
        <string>PIN :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_email">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>350</y>
         <width>350</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
border: 1px solid white;
border-radius:10px;
background-color: rgb(255, 233, 248);
}

</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_phone">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>320</y>
         <width>350</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
border: 1px solid white;
border-radius:10px;
background-color: rgb(255, 233, 248);
}

</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_name">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>290</y>
         <width>350</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
border: 1px solid white;
border-radius:10px;
background-color: rgb(255, 233, 248);
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_pin">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>380</y>
         <width>350</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
border: 1px solid white;
border-radius:10px;
background-color: rgb(255, 233, 248);
}

</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_type">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>260</y>
         <width>350</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox
{
color: rgb(0, 0, 0);
	
	background-color: rgb(255, 233, 248);
border: 1px solid white;

border-radius:10px;
}</string>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Savings </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Current </string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_join">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>440</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton

{
font: 10pt &quot;Segoe UI&quot;;
border-radius: 20px;
background-color: rgb(255, 0, 255);
color: rgb(255, 255, 255);
}

QPushButton:hover
{
background-color: rgb(170, 0, 255);
	
}</string>
       </property>
       <property name="text">
        <string>JOIN THE LEAGUE</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_back1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>460</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>600</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
color: rgb(255, 255, 255);
border: 0px;
background-color:none;
}

QPushButton:hover
{
color: rgb(255, 170, 255);
}</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
